### PR TITLE
Init Jenkins foundation for WebRTC Canary

### PR DESCRIPTION
**Pull request by lherman-cs**
Changes:
* Add README
* Add architectural diagrams
* Add Groovy scripts for seeding, orchestrator, and runner

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: webrtc-c/canary/jobs/runner.groovy [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+        stage('Preparation') {
+            steps {
+              echo params.toString()
+            }
+        }
+
+        stage('Build and Run') {
+            failFast true
+
+            parallel {
```
Comment (disa6302) [inline webrtc-c/canary/jobs/runner.groovy:106]: Maybe I missed it in the script, but, how are we ensuring the master and viewer run on different nodes?

Comment (lherman-cs) [reply webrtc-c/canary/jobs/runner.groovy:106]: It's defined using `MASTER_NODE_LABEL` and `VIEWER_NODE_LABEL`.

Viewer's agent will be set here, https://github.com/aws-samples/amazon-kinesis-video-streams-demos/pull/32/files/bc7fe786def5b88dc5592f1c4ed3f7581a7c70f0#diff-4358959bf091ec0bf5c150bca6fee305R116-R118

Master's agent will be set here, https://github.com/aws-samples/amazon-kinesis-video-streams-demos/pull/32/files/bc7fe786def5b88dc5592f1c4ed3f7581a7c70f0#diff-4358959bf091ec0bf5c150bca6fee305R77-R79

**Review by MushMal — APPROVED** [review]
I am not super familiar with the scripts but at a casual look seems ok

**Comment by disa6302** [comment]
The logic looks good to me. I guess we can always come back and change things if we need to

**Review by disa6302 — APPROVED** [review]

**Pull request merged**